.github/kokoro/ctest2junit.xsl

```diff
@@ -0,0 +1,120 @@
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">

```

Comment:
Forgive my ignorance, why do we need this file?

Reply:
Removed.